tests/unit_tests/test_wave_06.py

```diff
@@ -110,16 +107,14 @@ def test_swap_best_by_category():
         their_priority="Decor"
     )
 
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
-    # Assertions should check:
-    # - That the results is truthy
-    # - That tai and jesse's inventories are the correct length
-    # - That all the correct items are in tai and jesse's inventories, including the items which were swapped from one vendor to the other
+    assert result
+    assert len(tai.inventory) == 3
+    assert len(jesse.inventory) == 3
+    assert item_a not in tai.inventory
+    assert item_c in tai.inventory
+    assert item_f not in jesse.inventory
+    assert item_d in jesse.inventory
```

Comment:
Great start! It would also be good to check that the other items we expect in the inventories are there as well.